spec/requests/v1/tasks_spec.rb

```diff
@@ -194,14 +194,30 @@
       expect(json['data']['attributes']['status']).to eq 'completed'
     end
 
-    it 'records the user who submitted' do
+    it 'records the user who created the submission' do
       post "/v1/tasks/#{task.id}/no_business", headers: { 'X-Auth-Id' => user.auth_id }
 
       submission = task.submissions.last
 
       expect(submission.created_by).to eq(user)
     end
 
+    it 'records the user who completed the submission' do
+      post "/v1/tasks/#{task.id}/no_business", headers: { 'X-Auth-Id' => user.auth_id }
+
+      submission = task.submissions.last
+
+      expect(submission.submitted_by).to eq(user)
+    end
+
+    it 'records the time of submission' do
+      post "/v1/tasks/#{task.id}/no_business", headers: { 'X-Auth-Id' => user.auth_id }
+
+      submission = task.submissions.last
+
+      expect(submission.submitted_at).to be_present
```

Comment:
As above, would make for a better spec if it confirmed the value of the timestamp is within range.